src/test/regress/expected/multi_truncate.out

```diff
@@ -314,7 +324,7 @@ SELECT * FROM test_local_truncate;
 (1 row)
 
 -- Undistribute table
-SELECT master_drop_all_shards('test_local_truncate', 'pubic', 'test_local_truncate');
+SELECT master_drop_all_shards('test_local_truncate', 'public', 'test_local_truncate');
```

Comment:
I wanted to fix a typo in here, but as it turns out we have a bigger problem here.

The table `test_local_truncate` is in schema `multi_truncate` and not public. Does this mean that `master_drop_all_shards` does not respect the schema name parameter? If we had two distributed tables sharing the same name, but in different schemas, would it drop the shads of the wrong table?

Reply:
That's correct, it ignores the schema name and table name if the table still exists.

It uses the schema and table name only when the table does not exist, which is only the case in the SQL_DROP trigger.